db/migrate/20220201232357_create_reminders.rb

```diff
@@ -0,0 +1,12 @@
+class CreateReminders < ActiveRecord::Migration[6.1]
+  def change
```

Comment:
I can see that you used `rails g model Reminder` with subscription id as an integer. To avoid problems down the road, consider rolling back this migration and deleting/recreating the Reminders model with subscription as a reference. This will make life easier later. Subscriptions should `has_many` Reminders. See linked article for how to generate migration/model

https://stackoverflow.com/questions/8742633/rails-model-has-many-belongs-to-relations/8743097#8743097

(big difference being the subscription id is not an integer, it's a UUID!)